csfieldguide/chapters/content/en/human-computer-interaction/sections/interface-usability.md

```diff
@@ -104,13 +104,17 @@ It has English captions, but the most interesting part is what is happening in t
 ## Human short term memory
 
 Another important length of time to bear in mind is our *short term memory* time, which is usually a matter of seconds.
-To remember something for longer, the user needs to rehearse it (repeat it over) or make a note of the information, such as writing it down.
+To remember something for longer, the user needs to rehearse it (repeat it over) or make a note of the information, such as writing it down. Try the interactive below and see how many items you can remember without writing anything down; chances are you won't be able to remember them all!
```

Comment:
New line for new sentence. I'm not sure about the part after the semicolon, but don't have any suggestions yet